src/images/kafka.rs

```diff
@@ -0,0 +1,112 @@
+use crate::{core::WaitFor, Image};
+use std::collections::HashMap;
+
+const CONTAINER_IDENTIFIER: &str = "confluentinc/cp-kafka";
+const DEFAULT_TAG: &str = "6.1.1";
+
+pub const DEFAULT_BROKER_LISTENER_PORT: u16 = 9092;
+pub const DEFAULT_LOCALHOST_LISTENER_PORT: u16 = 19092;
+
+#[derive(Clone, Debug, Default)]
+pub struct KafkaArgs(pub Vec<String>);
```

Comment:
`KafkaArgs` should be a strongly-typed struct where we only convert to strings using `IntoIterator`. Checkout [this](https://github.com/testcontainers/testcontainers-rs/blob/df3805945219040d38c9112c0579b898249cdf3b/src/images/coblox_bitcoincore.rs#L100-L189) for some inspiration.

Reply:
Good point. Addressed in 0ae4d44.